src/quipucordsctl/constants.py

```diff
@@ -0,0 +1,23 @@
+"""Global constants for quipucordsctl."""
+
+# System commands commonly run
+SYSTEMCTL_USER_RESET_FAILED_CMD = ["systemctl", "--user", "reset-failed"]
+SYSTEMCTL_USER_DAEMON_RELOAD_CMD = ["systemctl", "--user", "daemon-reload"]
+SYSTEMCTL_USER_STOP_QUIPUCORDS_APP = ["systemctl", "--user", "stop", "quipucords-app"]
```

Comment:
I haven't been following quipucordsctl closely, but shouldn't that be branding-aware?

Reply:
That being said, as I think more about this new `constants` module, I wonder if these should go back to settings since they will likely need to be dynamic after DISCOVERY-1125. 🤔 Or we should in the future consider importing settings and injecting branding-aware names into some of these variables, but then they aren't really "constants", are they?

Basically, I'm wondering if the `constants` module is actually a good idea right now. I had similar thoughts previously when I was implementing the earlier commands, but I just threw everything _constant-like_ into the `settings` module because I knew that _one day soon_ (no later than DISCOVERY-1125) I or someone else would make changes to make many of them dynamically branding-aware.